samples/Exporters/Web/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
```

Comment:
appsettings.json already contain key, if we will put it in appsettings.Development.json - it will overrride it. Probably - it will be easier to drop environment specific config file

Reply:
deleted